src/app/containers/app/app.component.ts

```diff
@@ -9,13 +9,23 @@ import { Store } from '@ngrx/store';
   styleUrls: ['./app.component.scss'],
   encapsulation: ViewEncapsulation.None
 })
-export class AppComponent {
+export class AppComponent implements OnInit {
   constructor(
     private logger: LoggerService,
     private store: Store<fromActions.State>
   ) {
   }
 
+  ngOnInit() {
+    this.logger.trace('AppComponent: logger.trace()');
```

Comment:
this should be moved from here to <exui-logger> and component name pass as an input property

Reply:
Done